content/administration/upgrade.rst

```diff
@@ -1,226 +1,371 @@
-:show-content:
-
-.. |assistance-contact| replace::
-   If you need Odoo assistance on this matter, please get in touch with your Odoo Account Manager or
-   our `Sales department`_.
-.. _Sales department: mailto:sales@odoo.com
-
 =======
 Upgrade
 =======
 
-.. toctree::
-   :titlesonly:
+An upgrade is the process of moving your database from an older version to a newer :doc:`supported
+version <maintain/supported_versions>` (e.g., Odoo 14.0 to Odoo 16.0). Frequently upgrading is
+essential as each version comes with new and improved features, bug fixes, and security patches.
+
+.. _upgrade_faq/rolling_release:
+
+.. spoiler:: Automatic upgrades: Odoo Online's Rolling Release process
+
+   The Rolling Release process allows Odoo Online customers to upgrade their database directly from
+   a message prompt sent to the database administrator as soon as a new version is released. The
+   invitation to upgrade is only sent if no issues are detected during the automatic tests.
+
+   .. image:: upgrade/rr-upgrade-message.png
+      :alt: The upgrade message prompt on the top right of the database
+
+   It is strongly recommended to manually :ref:`test the upgrade first <upgrade/test_your_db>`.
+   Clicking :guilabel:`I want to test first` redirects to `the database manager
+   <https://www.odoo.com/my/databases/>`_, where it is possible to request an upgraded test database
+   and check it for any discrepancies.
 
-   upgrade/odoo_online
-   upgrade/odoo_sh
-   upgrade/on_premise
-   upgrade/faq
```

Comment:
These pages should each be replaced by a [redirect rule](https://github.com/odoo/documentation/blob/16.0/redirects/MANUAL.md).